app/Features.tsx

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import Image from "next/image";
+
+export default function Features() {
+  return (
+    <section>
+      {/* This is for the Features header */}
+      <div>
+        <div className="flex flex-col items-center justify-center mt-15 lg:mt-16">
+          <div>
+            <p className="w-44 font-sans text-xl text-center text-black tracking-widest sm:text-4xl lg:text-5xl">Features</p>
+          </div>
+          <div className="bg-[#191970] w-44 h-1 sm:w-96 lg:w-5/12"></div>
```

Comment:
dont use magic numbers or magic strings 

refer:
![image](https://github.com/ezerssss/unihub-web/assets/72904036/f9cd84de-e0fd-4398-8a38-65345bca2206)


to resolve put this in the tailwind config just how we have `bg-primary-100` you can do that for this color as well

Reply:
instead of a `div` use an `hr` instead